README.md

```diff
@@ -72,9 +82,29 @@ By default, the stack exposes the following ports:
 * 9300: Elasticsearch TCP transport
 * 5601: Kibana
 
-*WARNING*: If you're using *boot2docker*, you must access it via the *boot2docker* IP address instead of *localhost*.
+*WARNING*: If you're using `boot2docker`, you must access it via the `boot2docker` IP address instead of `localhost`.
+
+*WARNING*: If you're using *Docker Toolbox*, you must access it via the `docker-machine` IP address instead of `localhost`.
+
+# Create Indexes
+
+When Kibana launches, it will not contain any out-of-the-box indexes. Instead,
+you can run this command to create a Logstash index:
 
-*WARNING*: If you're using *Docker Toolbox*, you must access it via the *docker-machine* IP address instead of *localhost*.
+```bash
+$ curl -XPUT http://elasticsearch:9200/.kibana/index-pattern/logstash-* -d \
```

Comment:
Will not work from localhost because it can't resolve "elasticsearch". Maybe worth elaborating a little bit?